Postgres-install-instructions.md

```diff
@@ -30,43 +30,33 @@
 
 postgres will be running on port 5432
 
+## Creating Database
 
-## Setting up Users and Permissions
+Run the following in command line:
+`CREATEDB my_db;`
```

Comment:
this creates a db outside of using psql. the previous functions (i think?) worked